### PR TITLE
styling mostly, plus more validations

### PR DESCRIPTION
**Pull request by snugthejoiner**
@Brit200313 looking good now.

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -15,3 +15,4 @@
 every 1.day, :at => '9:40 pm' do
   rake "send_monthly", :environment => 'development'
```
Comment (Brit200313) [inline config/schedule.rb]: I'd like to see this development line take off before I merge, @snugthejoiner

File: app/controllers/personal_events_controller.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def create
        flash[:notice] = "Your event was saved."
        redirect_to @person
      elsif @personal_event.invalid?
-       flash[:alert] = "You must enter a description and a valid date! Nothing can be blank and the date of the event must be after the person was born."
+       flash[:alert] = "You must enter a description and a valid date! Nothing can be blank and the date of the event must be after the person was born (and before that person died, if applicable)."
```
Comment (Brit200313) [inline app/controllers/personal_events_controller.rb]: Try to trim this error down. It's super lengthy.

**Comment by snugthejoiner** [comment]
Lovely.

I'm at work and don't have my Macbook. But I realize now - I could `ssh` into DigitalOcean and do `git pull` to push changes to the production server. Then when I get home I could pull to my local master.

Any reason to **not** do that?

**Comment by Brit200313** [comment]
Nope! Go to town. Just remember your work laptop does not have ssh key
access.

On Tue, Sep 29, 2015 at 9:51 AM, snugthejoiner <notifications@github.com>
wrote:

> Lovely.
>
> I'm at work and don't have my Macbook. But I realize now - I could ssh
> into DigitalOcean and do git pull to push changes to the production
> server. Then when I get home I could pull to my local master.
>
> Any reason to *not* do that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/snugthejoiner/theusthread/pull/38#issuecomment-144067290>
> .
>



-- 
Brittany Martin
about.me/brittany.martin
sfviapgh.com

**Comment by snugthejoiner** [comment]
Very good. Haven't done it yet but I will

Here's a link to where you can preview and download my old band's new album called Serpent Umbrella, songs and singing by me:
http://thegravelpit.bandcamp.com/releases
pw: 56v5-68xh

**Comment by Brit200313** [comment]
Thanks so much!

**Pull request merged**